index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="ro">
+    <meta charset="UTF-8">
+    
+    <head class="header">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title> Save the planet</title>
+        <link href="./CSS/homestyle.css" rel=stylesheet>         
+    </head>
+    <body>
+        <section class="coverContainer">
+            <div class="coverImage">
+                <div class="shape"><img  src="./Photos/Shape.png" alt="shape"></div>
+                <div class="coverContent">
+                    <div class="subtitle">
+                        <div class="ractangleImg"><img  src="./Photos/Rectangle.png" alt="rectangle"></div>
+                        WE TEACH YOU TO PROTECT OUR   
```

Comment:
aici trebuie sa ii pui tag de titlu daca e titlu
w3schools.com/html/html_headings.asp
apoi ii pui clasa si o stilizezi "hero__title", mereu pui transform: uppercase in css daca vrei ca textul tau sa fie scris cu litere mari